api/src/main/java/keywhiz/api/model/Client.java

```diff
@@ -87,7 +87,7 @@ public Client(@JsonProperty("id") long id,
       @JsonProperty("enabled") boolean enabled,
       @JsonProperty("automationAllowed") boolean automationAllowed) {
     this.id = id;
-    this.name = checkNotNull(name);
+    this.name = checkNotNull(name, "Client name must not be null");
     this.description = nullToEmpty(description);
```

Comment:
This is functionally unrelated to the change, but I found the error messages helpful while debugging errors related to creating test data.